src/healthkit.cs

```diff
@@ -2432,6 +2538,15 @@ interface HKWorkoutRouteBuilder {
 
 		[Async, Wrap ("FinishRoute (workout, metadata != null ? metadata.Dictionary : null, completion)")]
 		void FinishRoute (HKWorkout workout, HKMetadata metadata, Action<HKWorkoutRoute, NSError> completion);
+
+		[Watch (5, 0), iOS (12, 0)]
+		[Async, Protected]
+		[Export ("addMetadata:completion:")]
+		void AddMetadata (NSDictionary metadata, Action<bool, NSError> completion);
+
+		[Watch (5, 0), iOS (12, 0)]
+		[Async, Wrap ("AddMetadata (metadata != null ? metadata.Dictionary : null, completion)")]
```

Comment:
No need to change anything, but for the future you can use `metadata?.Dictionary` to make the wrap expression smaller 😉